trade-finance-manager-ui/templates/_partials/footer.njk

```diff
@@ -4,14 +4,18 @@
       <div class="govuk-footer__meta">
         <div class="govuk-footer__meta-item govuk-footer__meta-item--grow">
           <h2 data-cy="contact-us-footer">Contact us</h2>
-
           <dl class="contact-details">
-            <dt>Email:</dt>
-            <dd>DigitalService.TradeFinance@ukexportfinance.gov.uk</dd>
+            {% if CONTACT_US_SELF_SERVICE_PORTAL_URL %}
```

Comment:
Is the plan to phase this out eventually? Should there be a ticket & `TODO` for this?

Reply:
So...it could technically be phased out later. I thought about it and maybe think it's better to keep in because:
- We're going to have different env vars for non dev and prod
- Therefore we are always going to have an env var for this
- Therefore there is a chance this isn't present
- Therefore we should allow a fallback to email address if not present
- As part of another ticket, we've updated the email addresses, so I've added in the same pattern here (with fallback, just in case!)

Does that make sense to you? Or should we make a ticket for it to scream if this env var is missing?

Reply:
I've added a ticket for this and for a review of whether we want to remove this and added comments

Reply:
Yeah think it feels weird to fail silently but makes sense to worry about that once Prod has the relevant env updated